test/intl402/DurationFormat/constructor/supportedLocalesOf/basic.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.DurationFormat.supportedLocalesOf
+description: Tests that Intl.DurationFormat has a supportedLocalesOf property, and it works as expected.
+features: [Intl.DurationFormat]
+---*/
+
+assert.sameValue(typeof Intl.DurationFormat.supportedLocalesOf, "function",
+                 "supportedLocalesOf should be supported.");
+
+const defaultLocale = new Intl.DurationFormat().resolvedOptions().locale;
+const notSupported = "zxx"; // "no linguistic content"
+const requestedLocales = [defaultLocale, notSupported];
+
+const supportedLocales = Intl.DurationFormat.supportedLocalesOf(requestedLocales);
+assert.sameValue(supportedLocales.length, 1, "The length of supported locales list is not 1.");
```

Comment:
These assertion messages seem unusual compared to what I've seen elsewhere in test262. I think "The length of the supported locales list should be 1" would be more usual. Same for the message just below this one.